config/manager/kustomization.yaml

```diff
@@ -16,5 +16,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: splunk/splunk-operator
+  newTag: 1.1.0
```

Comment:
Also, is this tag also correct? it should be something 2.x, isn't it? @vivekr-splunk ?

Reply:
yes it should be 2.0.0 but creation of image is done dynamically, just for the purpose of having right number we can change this to 2.0.0 . pipeline always uses right tag name when it creates

Reply:
Thanks Vivek and Gaurav. Updated to 2.0.0